watcher_fen.go

```diff
@@ -97,9 +98,9 @@ func (f *fen) Init() (err error) {
 }
 
 func fi2fo(fi os.FileInfo, p string) FileObj {
-	st, ok := fi.Sys().(*syscall.Stat_t)
+	st, ok := fi.Sys().(*unix.Stat_t)
 	if !ok {
-		panic(fmt.Sprintf("fen: type should be *syscall.Stat_t, %T instead", st))
+		panic(fmt.Sprintf("fen: type should be *unix.Stat_t, %T instead", st))
```

Comment:
Here should be `syscall` instead of `unix`.
If we are here already you could change `st` to `fi.Sys()`.